migrations/20200328131352_initial_schema.ts

```diff
@@ -5,7 +5,7 @@ export async function up(knex: Knex): Promise<void> {
     .createTable('tickets', table => {
       table.increments('id').primary()
 
-      table.string('description')
+      table.string('description', 500)
```

Comment:
These changes are generally better in a separate migration rather than modifying the existing one. This time it's ok since the app is in development, but once we move to prod these should be separated.

Reply:
You're right, I'mma create an issue for this instead.

Reply:
You don't have to, we already have #142